astropy/io/ascii/daophot.py

```diff
@@ -91,7 +101,24 @@ def get_cols(self, lines):
                         coldef_lines[i] = coldef_lines[i] + line_stripped
                         last_coldef_line[i] = line_stripped
                         break
-
+        
+        if self.unknown_apertures:
+            #determine the number of apertures in the record from the first data-containing line
+            #in the table given the width specifications in the header.
+            def first_true(iterable, default=False, pred=None):
```

Comment:
I don't know. I feel like I once made a very similar comment on some similar-looking code that you wrote?  Or it was just deja-vu for no reason.  Glitch in the matrix.